cmd/minikube/cmd/root_test.go

```diff
@@ -17,28 +17,17 @@ limitations under the License.
 package cmd
```

Comment:
changes to _root_test.go_ can be ignored. I extracted some testing framework code into _viper.go_, since I initially used it for _start_test.go_. In the end I changed my testing approach for _start_test.go_, though. 

I still find the extraction useful. It is really a no-op for now. If someone feels strongly about this, this is a change which could be reverted or extracted into a separate commit